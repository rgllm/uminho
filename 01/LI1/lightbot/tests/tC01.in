["aBaBa","aBaBa","aBaBa","BaBaB","BaBaB","BaBaB", "4 0 N", "LALALSAAESLEALALSAADSLDALALSAAESLEALALSAADSLDALAL"]