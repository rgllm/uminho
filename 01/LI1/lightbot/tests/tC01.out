["4 0","4 1","4 2","3 5","3 4","3 3","2 0","2 1","2 2","1 5","1 4","1 3","0 0","0 1","0 2","FIM 49"]